internal/annotations/annotations.go

```diff
@@ -434,9 +417,15 @@ func extractCommaDelimitedStrings(s string) []string {
 	// Split by comma.
 	out := strings.Split(s, ",")
 
-	// Trim spaces in place.
+	// Apply trimming and sanitization functions in place.
 	for i := range out {
+		// Trim spaces.
 		out[i] = strings.TrimSpace(out[i])
+
+		// Apply optional sanitization functions (e.g. upper-casing).
+		for _, sanitizeFn := range sanitizeFns {
+			out[i] = sanitizeFn(out[i])
+		}
```

Comment:
nit: Filtering of empty strings 431-434 can be done as part of this loop

Reply:
e590eac